aspnetcore/signalr/client-features.md

```diff
@@ -8,25 +8,41 @@ ms.date: 11/12/2019
 no-loc: [Blazor, "Identity", "Let's Encrypt", Razor, SignalR]
 uid: signalr/client-features
 ---
-# ASP.NET Core SignalR client features
+# ASP.NET Core SignalR Clients
+
+## Versioning, Support and Compatibility
+
+The SignalR clients ship alongside the server components and are versioned to match. Any supported client can safely connect to any supported server, and any compatibility issues would be considered bugs to be fixed. SignalR clients are supported in the same support lifecycle as the rest of .NET Core. See [the .NET Core Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core) for details.
+
+Many features require a compatible client **and** server. See below for a table showing the minimum versions for various features.
+
+The 1.x versions of SignalR map to the 2.1 and 2.2 .NET Core releases and have the same lifetime. For version 3.x and above, the SignalR version exactly matches the rest of .NET and has the same support lifecycle.
+
+| SignalR Version | .NET Core Version | Support Level | End of Support |
+| - | - | - | - |
+| 1.0.x | 2.1.x | Long Term Support | August 21, 2021 |
+| 1.1.x | 2.2.x | End Of Life | December 23, 2019 |
+| 3.x or higher | *same as SignalR version* | See the [the .NET Core Support Policy](https://dotnet.microsoft.com/platform/support/policy/dotnet-core)
+
+**NOTE:** In ASP.NET Core 3.0, the JavaScript client *moved* to the `@microsoft/signalr` NPM package.
```

Comment:
```suggestion
**NOTE:** In ASP.NET Core 3.0, the JavaScript client *moved* to the `@microsoft/signalr` npm package.
```

Reply:
We were just talking about the confusion caused by still having a non-deprecated `@aspnet/signalr` npm package in triage when developers working on new SignalR apps will want to use `@microsoft/signalr`.

Would we be okay with deprecating `@aspnet/signalr` if we verify the latest `@microsoft/signalr` client is compatible with 2.1.x SignnalR Core servers? There would still be the `signalr` npm package creating some confusion, but I'm not sure how we can fix that short of better package descriptions.

Reply:
I would be perfectly stoked if we could work it out so that the 2.1 and 3.1 server components both worked with the 3.1 client, and then we could deprecate. Huge fan.

Reply:
> I would be perfectly stoked if we could work it out so that the 2.1 and 3.1 server components both worked with the 3.1 client

We already make that commitment. I'm more concerned about the folks who are on the `@aspnet/signalr` 1.0 client **because it is itself a long-term servicing release** and they don't want to upgrade yet. Having a message display whenever they `npm install` telling them they are using a deprecated package is inaccurate and conflicts with our published support policy. The 1.0 client is supported until 2021 according to our support policy.